Henfam/lib/models/menu_item.dart

```diff
@@ -1,12 +1,36 @@
+import 'package:Henfam/models/menu_modifier.dart';
+import 'package:Henfam/models/models.dart';
 import 'package:equatable/equatable.dart';
 
 class MenuItem extends Equatable {
   final String name;
+  final String description;
   final double price;
-  final List<MenuItem> addOns;
+  final List<String> modifiers;
+  final List<ModifierItem> modifiersChosen;
 
-  MenuItem(this.name, this.price, this.addOns);
+  MenuItem(this.name, this.description, this.price, this.modifiers,
+      {this.modifiersChosen});
 
   @override
-  List<Object> get props => [name, price, addOns];
+  List<Object> get props => [
+        name,
+        description,
+        price,
+        modifiers,
+      ];
+
+  void setModifiersChosen(List<ModifierItem> selectedItems) {
+    modifiersChosen.clear();
+    modifiersChosen.addAll(selectedItems);
+  }
+
+  List<ModifierItem> cloneModifiersChosen() {
```

Comment:
Why do we need this function? I don't understand why we need to do a deep copy on the modifiers

Reply:
Oh is this a version of a getter for ModifierItem?